src/components/EventCard.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { ReactElement } from "react";
+import ".././styles.css";
+import PinSVG from ".././public/Pin.svg";
+import TextIconSVG from ".././public/TextIcon.svg";
+import MeatballMenuSVG from ".././public/MeatballMenu.svg";
+
+export interface EventCardProps {
+  startTime: Date;
+  endTime?: Date;
+  name: string;
+  location: string;
+  description: string;
+}
+
+/**
+ * Renders a single event in the feed
+ */
+const EventCard = ({
+  startTime,
+  endTime,
+  name,
+  location,
+  description,
+}: EventCardProps): ReactElement => {
+  let startTimeString: string = startTime.toLocaleString("en-US", {
+    hour: "numeric",
+    minute: "numeric",
+    hour12: true,
+  });
+  let endTimeString: string | undefined = endTime
+    ? endTime.toLocaleString("en-US", {
+        hour: "numeric",
+        minute: "numeric",
+        hour12: true,
+      })
+    : undefined;
+
+  return (
+    <>
+      <div className="pl-4 pt-4 pr-6 w-4/5">
+        <span className="font-sans">
+          {startTimeString + (endTime ? " to " + endTimeString : "")}
+        </span>
+        <div className="flex flex-row justify-between items-start mb-4">
+          <div className="not-italic font-bold text-2xl leading-8 font-sans break-words w-4/5">
+            {name}
+          </div>
+          <img
+            src={MeatballMenuSVG}
+            alt="Menu svg"
+            aria-label="Open Event Card details"
+          />
+        </div>
+        <div className="flex flex-row items-start mb-4">
+          <img src={PinSVG} alt="Pin svg" className="p-1 pt-0" />
+          <span className="text-body-mobile pl-2 pr-12 pt-0.5 font-montserrat break-words w-full">
+            {location}
+          </span>
+        </div>
+        <div className="flex flex-row items-start mb-6">
+          <img src={TextIconSVG} alt="TextIcon svg" className="p-1 pt-0" />
+          <p className="text-body-mobile pl-2 pr-12 font-montserrat break-words w-full">
+            {description}
+          </p>
+        </div>
+
+        <div className="flex flex-row justify-between col-gap-4">
+          <div className="button-base-white px-2 my-2">Unregister</div>
```

Comment:
If possible, I'd put another div container around these buttons and add a margin on that container for more control over the space between the buttons. I think at the moment the `justify-between` on the flexbox container might look odd on larger screen sizes like this: 

![image](https://user-images.githubusercontent.com/40529597/199829877-a30ba6e7-5617-451e-b06d-3b58e77f5ccd.png)

And I realized Tailwind CSS doesn't support having gaps for flexboxes, so I don't think `col-gap-4` will work. Instead I'd recommend adding margin spacing to a parent container for each of these buttons!

Reply:
Mm yeah good call! I don't remember why I put `col-gap-4` initially, for some reason I thought I remembered it being a thing and even searching it up?? mandela effect go crazy

Would you mind explaining why we want to put the margins on the buttons' parent div instead of the buttons themselves?

Reply:
Omg good catch - margins on the buttons itself should work! I'm not sure why but when I was testing things on the preview, I was modifying the `padding` instead of the `margin` and was wondering why the button wasn't giving itself a margin, so I thought it needed a parent div with a margin instead. tysm for the catch!

Reply:
ok!! going to give the buttons margins then and remove the parent divs